### PR TITLE
Add  support for MFannot

### PR DESCRIPTION
**Pull request by natacha-beck**

### CODE REVIEW

File: userfiles/FASTA/fasta.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#
```
Comment (prioux) [inline userfiles/FASTA/fasta.rb:1]: The subdirectory 'fasta' should be in lowercase, following the Ruby convention.

**Comment by natacha-beck** [comment]
I put FASTA due to the naming of the format according to Wikipedia, but I can rename it.

File: userfiles/sequin_tbl/sequin_tbl.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+# Generic model for Tbl Sequin files.
+class TblSequin < CSVFile
```
Comment (prioux) [inline userfiles/sequin_tbl/sequin_tbl.rb:23]: The class name is inconsistent with the path to the ruby file (sequin_tbl/sequin_tbl.rb)

**Review by prioux — APPROVED** [review]
We should consider adding some of the Boutiques modules directives at the end.

**Pull request merged**